types/react-jss/package.json

```diff
@@ -1,6 +1,8 @@
 {
   "private": true,
   "dependencies": {
-    "csstype": "^2.0.0"
+    "csstype": "^2.0.0",
+    "indefinite-observable": "^1.0.1",
+    "typescript": "^3.0.3"
```

Comment:
Sorry, dont have typescript installed globally so added it to run `tsc` from the package directory.

Will remove the dependency - is there a best practice? All the documentation said was `Otherwise, run tsc in the package directory`

Reply:
There may be a trick for having a local installation without a package.json entry but I, unfortunately, am not enough of an npm expert to know it.